### PR TITLE
Fix bugs introduced in #4959

### PR DESCRIPTION
**Pull request by themylogin**
@yocalebo what did you intended to do with https://github.com/freenas/freenas/pull/4959/files#diff-c44b6d54b740097b6b338bb38f20ffd3L1004 ? It broke on FreeNAS where we don't have failover plugin

### CODE REVIEW

File: src/middlewared/middlewared/plugins/network.py [hunk 1]
```diff
@@ -1001,7 +1001,7 @@ async def _common_validation(self, verrors, schema_name, data, itype, update=Non
                             'VLAN MTU cannot be bigger than parent interface.',
                         )
 
-        if not await self.middleware.call('failover.licensed'):
+        if await self.middleware.call('system.is_freenas') or not await self.middleware.call('failover.licensed'):
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/network.py:1004]: failover plugin should always be available in 12, no?

Comment (william-gr) [reply src/middlewared/middlewared/plugins/network.py:1004]: Ah, this is 11.3, ooops

**Comment by themylogin** [comment]
backport

**Comment by themylogin** [comment]
backport

**Pull request merged**